README.md

```diff
@@ -851,25 +851,25 @@ Example: `set_global_routing_layer_pitch Metal6 1.34`.
 
 ```
 set_clock_routing [-clock_pdrev_fanout fanout] \
-                  [-clock_topology_priority priority]
+                  [-clock_pdrev_alpha alpha]
```

Comment:
again, no point in putting -clock in the arg name for a command named set_clock_